src/components/admin/HistoryViewer/CommitViewer.tsx

```
@@ -0,0 +1,356 @@
+                        ) : (
+                            fileChanges.map((file) => (
+                                <button
+                                    key={file.filename}
+                                    onClick={() => setSelectedFilename(file.filename)}
+                                    className={cn(
+                                        "w-full flex items-center gap-2 px-3 py-2 text-sm rounded-md transition-all text-left group",
+                                        selectedFilename === file.filename
+                                            ? "bg-primary/10 text-primary font-medium"
+                                            : "text-muted-foreground hover:bg-muted hover:text-foreground"
+                                    )}
+                                >
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)